### DISCUSSION TITLE
Is there a way to stop http_port server in CH?

### DISCUSSION THREAD

**Discussion by athree** [ClickHouse/ClickHouse#50932] [Q&A] [answered]
Is there an interactive way to stop http_port server while CH running.

such as system stop http_server ?  i have not found in docs and code.

ths

**Comment by athree** [comment]
or i just want to reject xxx/ping access.

**Reply by UnamedRus** [reply]
https://docs.altinity.com/operationsguide/security/clickhouse-hardening-guide/network-hardening/#reduce-exposure

**Comment by DanRoscigno** [comment]
If you want to stop ClickHouse from opening the HTTP port then you have to add a file to config.d/ and disable it.  You cannot stop it without stopping the clickhouse-server process.

If you want to reject ICMP pings to the server, then you have to use iptables or other tools, and if you are doing that, then you can also restrict HTTP access that way also.

**Comment by athree** [comment, marked as answer]
ok， i got. 

In my case, i just want to disable one http_port of two starting CH instances which both configed reuse_port.

And it seems there is no exist way to do this, i have modified some code to do this.
